public/app/features/dashboard/services/TimeSrv.ts

```diff
@@ -56,6 +56,17 @@ export class TimeSrv {
 
     this.initTimeFromUrl();
     this.parseTime();
+    const range = rangeUtil.convertRawToRange(this.time, this.dashboard ? this.dashboard.getTimezone() : undefined);
+
+    if (range.to.isBefore(range.from)) {
```

Comment:
@fabio-silva Shouldn't this come after line 72, since that one will clone the update time, not the original one?

Reply:
well spotted, thanks 👍